scripts/CEE/kubelet-restart/script.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -e
```

Comment:
```suggestion
set -o errexit
```

...if you want it to match :D

Reply:
Done! Thanks :)